config/rbac/role.yaml

```diff
@@ -21,6 +21,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - serviceaccounts/token
+  verbs:
+  - create
```

Comment:
this is pretty sensitive and in past we've done other rbac roles for it specifically... we may want to consider that here too (similar to being able to get all Secrets in a cluster)...

Reply:
sounds good! let me make this a separate file with markers and we can have the user decide if they want to deploy. For helm we can make it conditional based on the `tokenRequests` field.

Reply:
done!